### PR TITLE
[F#/Oxpecker] Improved "update" and "fortunes" benchmarks

### PR DESCRIPTION
**Pull request by Lanayx**

### CODE REVIEW

File: frameworks/FSharp/oxpecker/src/App/Db.fs [hunk 1]
```diff
@@ -95,15 +100,15 @@ module Db =
...
+            command.Parameters.Add(NpgsqlParameter<int>(
+                ParameterName = rnParamName, DbType = DbType.Int32, TypedValue = randomNumber)) |> ignore
+            command.Parameters.Add(NpgsqlParameter<int>(
+                ParameterName = idParamName, DbType = DbType.Int32, TypedValue = results[i].id)) |> ignore
             results[i] <- { results[i] with randomnumber = randomNumber }
 
     let doMultipleUpdates (count: int) =
         task {
-            use conn = new NpgsqlConnection(ConnectionString)
+            use conn = new NpgsqlConnection(MultiplexedConnectionString)
```
Comment (sebastienros) [inline frameworks/FSharp/oxpecker/src/App/Db.fs:111]: Did you find out that multiplexing was detrimental in Fortunes (we did too, for these machines at least)

Comment (Lanayx) [reply frameworks/FSharp/oxpecker/src/App/Db.fs:111]: Yes, I did, you are right

**Comment by Lanayx** [comment]
@NateBrady23 Hi! More than 2 weeks have passed without merge, are there any problems with my PR?

**Comment by Lanayx** [comment]
Squashed commits into one, hopefully this will help merging...

**Pull request merged**